client/src/components/History/CurrentHistory/HistoryDropZone.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
Not requesting a change for this, but splitting this one out is overkill.  It'd be different if there was component logic, or if it was reused, but for a single element and css rule I'd rather see this inline in the parent component.

Reply:
Yea, I agree. Had a similar discussion and then left it in, as separat component, for now since there might be additions to it soon.